ramls/rtypes/collection-with-json-response.raml

```diff
@@ -0,0 +1,94 @@
+#%RAML 1.0 ResourceType
```

Comment:
Can this file be here or should a PR be in this https://github.com/folio-org/raml project?

Reply:
It would be better to have these files in raml project as they are already used in mod-orders and can be potentially useful for all business modules.
Prepared PR to add this rTypes to ramls https://github.com/folio-org/raml/pull/123